test/TestTaskIncludes.py

```diff
@@ -56,3 +56,10 @@ def test_include_tasks_in_role(self):
         runner = Runner(self.rules, filename, [], [], [])
         runner.run()
         self.assertEqual(len(runner.playbooks), 4)
+
+    @unittest.skipIf(parse_version(ansible.__version__) < parse_version('2.7'), "not supported with ansible < 2.7")
```

Comment:
Since we're moving towards pytest-based tests, we should avoid using `unittest` module.
Here's an example of a test module using only `pytest`: https://github.com/ansible/ansible-lint/blob/41d597d0583967fc21c0f1b5a1281b4205b14d37/test/TestIncludeMissingFileRule.py.

Reply:
@webknjaz Thanks for helping with that. I still see few conflicts.

Reply:
Yeah, I was doing all the coding in the browser. This needs to be rebased manually. But also we need other comments to be addressed. So maybe if @patrickjahns takes over, he'll do this in his PR. I only committed this change to avoid going back and forth with pytest coding style and people being unfamiliar w/ it. Other things like almost missing test coverage would still have to be fixed by somebody.

Reply:
Editing PRs in browser, that is a brave task for anything more than a one liner.